pkg/loadbalancer/loadbalancer.go

```diff
@@ -475,17 +475,26 @@ func (a *L3n4Addr) DeepCopy() *L3n4Addr {
 	}
 }
 
-// Hash calculates L3n4Addr's internal SHA256Sum.
+// Hash calculates a unique string of the L3n4Addr e.g for use as a key in maps
 func (a L3n4Addr) Hash() string {
+	const lenIPv4 = 15
+	const lenProto = 1
+	const lenPort = 6
+	const lenScope = 2
```

Comment:
`const lenScope = 1` // len("0") or len("1"), there isn't a "|" as part of the scope

Reply:
The sum should be correct. I did not add the `|` to the IP address, and added it to the scope instead